backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -22,6 +22,8 @@
 import yaml
 from kubernetes import client
 from kubernetes.client import exceptions
+from openshift.dynamic import DynamicClient
+from openshift.dynamic.exceptions import ResourceNotFoundError
```

Comment:
Please adjust the imports here:
- `from openshift import dynamic` and then use `dynamic.DynamicClient`
- `from openshift.dynamic import exceptions as dynamic_exceptions` and then use `dynamic_exceptions.ResourceNotFoundError`